src/sentry/replays/endpoints/organization_replay_details.py

```diff
@@ -1,26 +1,48 @@
 import uuid
 
+from drf_spectacular.utils import extend_schema
 from rest_framework.request import Request
 from rest_framework.response import Response
 
 from sentry import features
 from sentry.api.base import region_silo_endpoint
 from sentry.api.bases.organization import NoProjects, OrganizationEndpoint
+from sentry.apidocs.constants import RESPONSE_BAD_REQUEST, RESPONSE_FORBIDDEN
+from sentry.apidocs.examples.replay_examples import ReplayExamples
+from sentry.apidocs.parameters import GlobalParams
+from sentry.apidocs.utils import inline_sentry_response_serializer
 from sentry.constants import ALL_ACCESS_PROJECTS
 from sentry.models.organization import Organization
-from sentry.replays.post_process import process_raw_response
+from sentry.replays.post_process import ReplayDetailsResponse, process_raw_response
 from sentry.replays.query import query_replay_instance
+from sentry.replays.validators import ReplayValidator
 
 
 @region_silo_endpoint
+@extend_schema(tags=["Replays"])
 class OrganizationReplayDetailsEndpoint(OrganizationEndpoint):
     """
     The same data as ProjectReplayDetails, except no project is required.
     This works as we'll query for this replay_id across all projects in the
     organization that the user has access to.
     """
 
+    public = {"GET"}
+
+    @extend_schema(
+        operation_id="Retrieve a single replay instance",
```

Comment:
We should capitalize endpoint names. I just realized the crons endpoint aren't capitalized so I'll fix those too
```suggestion
        operation_id="Retrieve a single replay instance",
```

Reply:
got it, updating!

Reply:
oh wait sorry I have some other small things, I always forget to comment in batches :sweat_smile:

Reply:
give me a sec